pyTigerGraph/gds/gsql/dataloaders/hgt_loader.gsql

```diff
@@ -72,20 +79,40 @@ CREATE QUERY hgt_loader_{QUERYSUFFIX}(
         # Get seeds
         IF input_vertices.size()==0 THEN
             start = {seed_types};
+            HeapAccum<ID_Tuple> (1, tmp_id ASC) @@batch_heap;
+            @@batch_heap.resize(batch_s);
             IF filter_by IS NOT NULL THEN
+                FOREACH iter IN RANGE[0,1-1] DO 
```

Comment:
Is this number 1 (and 10 below) a placeholder?

Reply:
I caught this as well as I was converting other queries. It should be the variable corresponding to the number of heap inserts, to be parameterized.